app/screens/get-started-screen/device-account-modal.tsx

```diff
@@ -93,7 +93,7 @@ export const DeviceAccountModal: React.FC<DeviceAccountModalProps> = ({
         method: "POST",
         headers: {
           Authorization: `Basic ${auth}`,
-          Appcheck: appCheckToken,
+          Appcheck: appCheckToken || "undefined",
```

Comment:
```suggestion
          Appcheck: appCheckToken || undefined,
```

Reply:
it's a bit ugly but also intentional. the header need a string. generally no "undefined" should be passed but for type reason we need to enter something. oathkeeper would decline the call obviously if this is sent, but that's ok.